.pre-commit-config.yaml

```diff
@@ -35,7 +35,8 @@ repos:
             git archive HEAD \
               --prefix=apache-arrow-java/ \
               --output=apache-arrow-java.tar.gz && \
-              dev/release/run_rat.sh apache-arrow-java.tar.gz"
+              dev/release/run_rat.sh apache-arrow-java.tar.gz && \
+              rm -f apache-arrow-java.tar.gz"
```

Comment:
I guess it will get left behind if run_rat fails, but I suppose that's ok?

Reply:
Yes. In the case, we should check `apache-arrow-java.tar.gz`.